src/buttons/Button.tsx

```diff
@@ -50,10 +51,11 @@ export type ButtonProps = TouchableOpacityProps &
     disabledStyle?: StyleProp<ViewStyle>;
     disabledTitleStyle?: StyleProp<TextStyle>;
     raised?: boolean;
-    theme?: Theme;
   };
 
-const Button: React.FunctionComponent<ButtonProps> = (props: ButtonProps) => {
+const Button: React.FunctionComponent<ButtonProps> = (
```

Comment:
I think you accidentally added this to the prop type instead of the component type. Or it atleast should be added to both.